src/main/java/frc/robot/subsystems/localization/CameraIOPhoton.java

```diff
@@ -92,6 +94,19 @@ public void updateInputs(CameraIOInputs inputs) {
                 });
     }
 
+    private static PhotonPipelineResult filterTargets(PhotonPipelineResult unfiltered) {
+        List<PhotonTrackedTarget> targets = unfiltered.getTargets();
+        List<PhotonTrackedTarget> filteredTargets = new ArrayList<PhotonTrackedTarget>();
+
+        for (PhotonTrackedTarget target : targets) {
+            if (target.getPoseAmbiguity() < VisionConstants.maximumAmbiguity
+                    && target.getPitch() < VisionConstants.maximumPitch) {
```

Comment:
Can we use an absolute value so that if pitch is too positive or too negative it'll reject both?

Reply:
Also is there any reason to do this here and not in the filterPhotonPose method (or we could do it in both places)?
I'm not necessarily asking for you to change it, I'm just curious about the reasoning behind putting it here vs. somewhere else.